components/eamxx/src/physics/mam/eamxx_mam_microphysics.cpp

```diff
@@ -60,10 +62,19 @@ void MAMMicrophysics::set_grids(const std::shared_ptr<const GridsManager> grids_
   add_field<Required>("pseudo_density", scalar3d_layout_mid, q_unit, grid_name); // pdel
   add_field<Required>("cldfrac_tot", scalar3d_layout_mid, nondim, grid_name);
 
-  // tracer group (stores all aerosol prognostics)
-  add_group<Updated>("tracers", grid_name, Bundling::Required);
+  // aerosol tracers of interest: mass (q) and number (n) mixing ratios
+  add_field<Updated>("q_aitken_so4", scalar3d_layout_mid, q_unit, grid_name, "tracers");
+  add_field<Updated>("n_aitken_so4", scalar3d_layout_mid, n_unit, grid_name, "tracers");
+
+  // aerosol-related gases: mass mixing ratios
+  add_field<Updated>("q_soag", scalar3d_layout_mid, q_unit, grid_name, "tracers");
+  add_field<Updated>("q_h2so4", scalar3d_layout_mid, q_unit, grid_name, "tracers");
+
+  // Tracer group -- do we need this in addition to the tracers above?
+  add_group<Updated>("tracers", grid_name, 1, Bundling::Required);
```

Comment:
It doesn't look like you're using tracers... You could add the 4 fields above to a new group, and call it `aero_tracers`. I'd be interested in whether the AD can correctly bundle those 4. So far, we only have one group, the "tracers" group. This would be the first instance of a nested group of tracers being bundled _within_ the tracers group.

Reply:
You can, of course, still add these to the tracers group as well. You can include a field in as many groups as you want. So you can do
```
add_field<Updated>("q_aitken_so4", scalar3d_layout_mid, q_unit, grid_name, {"tracers","aero_tracers"});
...
add_group<Updated>("aero_tracers", grid_name, 1, Bundling::Required);
```

To be clear, the latter is really only important if a) you want to handle any aero tracer that may be declared by other atm procs, or b) you want them bundled (perhaps so you can loop over them and do some common ops). If you _expect_ that the only aero tracers are those declared in this class _and_ you don't need the aero tracers to be bundled, then you don't really need the group, and can simply work with the individual fields.